modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/validation/ParameterModelValidator.java

```diff
@@ -111,6 +124,31 @@ public void onParameter(ParameterizedModel owner, ParameterModel model) {
         validateNameCollisionWithTypes(model, ownerName, ownerModelType, extensionModelName,
                                        owner.getParameterModels().stream().map(p -> hyphenize(p.getName())).collect(toList()));
       }
+
+      @Override
+      public void onOperation(HasOperationModels owner, OperationModel model) {
+        model.getParameterModels().forEach(parameterModel -> parameterModel.getModelProperty(ImplementingTypeModelProperty.class)
```

Comment:
IMplementingTypeModelProperty o n a parameter??? where did you get that one from?

I think that you should just filter the parameters which metadata type has JAVA format, and then test the associated type. ImplementingTypeModelProperty makes no sense for a parameter